src/task_2.1/index.ts

```diff
@@ -0,0 +1,111 @@
+import express, { Request, Response, NextFunction } from 'express';
+import uuid from 'uuid/v1';
+import sortBy from 'lodash.sortby';
+import * as Joi from '@hapi/joi';
+import { createValidator } from 'express-joi-validation'
+import UserModel from './models/UserModel';
+import { PORT } from './constants';
+import { userCreateValidationSchema, userUpdateValidationSchema } from './validation';
+
+const users: UserModel[] = [];
+
+const validator = createValidator()
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }))
+
+app.param('id', (req: Request, res: Response, next: NextFunction, value) => {
+    
```

Comment:
Did you mean user extraction or anything else in this middleware? What is it for?

Reply:
Yes, my intention was to extract user Id passed as a parameter